android/src/app/organicmaps/widget/placepage/PlacePageController.java

```diff
@@ -76,6 +78,35 @@ public class PlacePageController implements Initializable<Activity>,
 
   private ValueAnimator mCustomPeekHeightAnimator;
 
+  class DefaultBottomSheetCallback extends BottomSheetBehavior.BottomSheetCallback
+  {
+    @Override
+    public void onStateChanged(@NonNull View bottomSheet, int newState)
+    {
+      Logger.d(TAG, "State change, new = " + PlacePageUtils.toString(newState));
+      if (PlacePageUtils.isSettlingState(newState) || PlacePageUtils.isDraggingState(newState) ||
+          mMwmActivity.isFinishing() || mMwmActivity.isDestroyed())
+        return;
+
+      PlacePageUtils.moveViewportUp(mPlacePage, mViewportMinHeight);
+
+      if (PlacePageUtils.isHiddenState(newState))
+      {
+        onHiddenInternal();
+      }
+    }
+
+    @Override
+    public void onSlide(@NonNull View bottomSheet, float slideOffset)
```

Comment:
What causes the call of this callback when activity is destroyed? Can we unsubscribe from it/cancel it when the app is closed, and avoid its call at all?

Reply:
@rtsisyk it was closed almost a year ago. Do we use the latest dependency version with that fix?